### PR TITLE
viddy 0.3.0 (new formula)

### PR DESCRIPTION
**Pull request by sachaos**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/viddy.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  head "https://github.com/sachaos/viddy.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.version=v#{version}")
+  end
+
+  test do
+    system "#{bin}/viddy", "-h"
```
Comment (carlocab) [inline Formula/viddy.rb:16]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Comment (sachaos) [reply Formula/viddy.rb:16]: @carlocab Thank you for your review!
Unfortunately, viddy is a TUI app and currently requires user input.
If a feature or option is added that does not require user input, I would like to write a test using it, but it is currently difficult.

Comment (carlocab) [reply Formula/viddy.rb:16]: Can we simulate pty input using Ruby's pty module?

Comment (sachaos) [reply Formula/viddy.rb]: @carlocab Thank you! I've added a test by using Ruby's pty module.

**Review by carlocab — APPROVED** [review]
Thanks, @sachaos!

**Pull request closed without merging**